collectors/ebpf.plugin/README.md

```diff
@@ -8,51 +8,407 @@ sidebar_label: "eBPF"
+-   ipc shm
+    -   Number of calls to `shm_get`. (`shmget_call`)
+    -   Number of calls to `shm_at`. (`shmat_call`)
+    -   Number of calls to `shm_dt`. (`shmdt_call`)
+    -   Number of calls to `shm_ctl`. (`shmctl_call`)
+
+### `[ebpf programs]` configuration options
+
+The eBPF collector enables and runs the following eBPF programs by default:
+
+-   `fd` :  This eBPF program creates charts that show information about calls to open files.
+-   `mount`: This eBPF program creates charts that show calls to syscalls mount(2) and umount(2).
+-   `shm`: This eBPF program creates charts that show calls to syscalls shmget(2), shmat(2), shmdt(2) and shmctl(2).
+-   `sync`: Monitor calls to syscalls sync(2), fsync(2), fdatasync(2), syncfs(2), msync(2), and sync_file_range(2).
+-   `network viewer`: This eBPF program creates charts with information about `TCP` and `UDP` functions, including the
+    bandwidth consumed by each.
+-   `vfs`: This eBPF program creates charts that show information about VFS (Virtual File System) functions.
+-   `process`: This eBPF program creates charts that show information about process life. When in `return` mode, it also
+    creates charts showing errors when these operations are executed.
+-   `hardirq`: This eBPF program creates charts that show information about time spent servicing individual hardware
+    interrupt requests (hard IRQs).
+-   `softirq`: This eBPF program creates charts that show information about time spent servicing individual software
+    interrupt requests (soft IRQs).
+-   `oomkill`: This eBPF program creates a chart that shows OOM kills for all applications recognized via
+    the `apps.plugin` integration. Note that this program will show application charts regardless of whether apps
+    integration is turned on or off.
+
+You can also enable the following eBPF programs:
+
+-   `cachestat`: Netdata's eBPF data collector creates charts about the memory page cache. When the integration with
+    [`apps.plugin`](/collectors/apps.plugin/README.md) is enabled, this collector creates charts for the whole host _and_
+    for each application.
+-   `dcstat` : This eBPF program creates charts that show information about file access using directory cache. It appends
+    `kprobes` for `lookup_fast()` and `d_lookup()` to identify if files are inside directory cache, outside and files are
+    not found.
+-   `disk` : This eBPF program creates charts that show information about disk latency independent of filesystem.
+-   `filesystem` : This eBPF program creates charts that show information about some filesystem latency.
+-   `swap` : This eBPF program creates charts that show information about swap access.
+-   `mdflush`: This eBPF program creates charts that show information about
+    multi-device software flushes.
+
+### Configuring eBPF threads
+
+You can configure each thread of the eBPF data collector. @thiagoftsm Can you give some details why/ when you should configure a thread? 
```

Comment:
Looks like we have a comment that snuck in to the main text. 

```suggestion
You can configure each thread of the eBPF data collector. @thiagoftsm Can you give some details why/ when you should configure a thread? 
```

Reply:
Good catch! 
@thiagoftsm Could you have a look?